src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -116,8 +117,12 @@ class SeekBar extends Slider {
     }
 
     if (liveTracker && liveTracker.seekableEnd() === Infinity) {
-      this.disable();
-    } else {
+      if (this.player_.readyState() === 0) {
```

Comment:
We could also check if `player.seekable().length` is non-zero rather than checking `player.readyState()`. Not sure which is better but `player.seekable().length` seems more straightforward to me. Regardless of what we do this section of code could use a comment.